pyartcd/pyartcd/pipelines/golang_builder_shipment.py

```
@@ -0,0 +1,371 @@
+            synopsis=f"Golang builder image update for OpenShift {self.ocp_version}",
+            topic=(
+                f"An update for the golang builder images is now available for "
+                f"Red Hat OpenShift Container Platform {self.ocp_version}."
+            ),
+            description=(
+                f"This update provides rebuilt golang builder images for "
+                f"Red Hat OpenShift Container Platform {self.ocp_version}.\n\n"
+                f"Golang group: {self.golang_group}"
+            ),
+            solution=(
+                f"The golang builder images are available from "
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)